### ISSUE TITLE
keybindings prevTab/nextTab with ctrl+pageup/ctrl+pagedown jumps to unexpected tabs

### ISSUE THREAD

**Issue by WilliamKyle** [microsoft/terminal#8233]
# Environment

```none
Windows build number: [run `[Environment]::OSVersion` for powershell, or `ver` for cmd]
Microsoft Windows [版本 10.0.19042.630]

Windows Terminal version (if applicable):
Windows Terminal
版本: 1.4.3141.0

Any other software?
```

# Steps to reproduce

* Add keybindings in settings
``` json
"keybindings":
    [
        { "command": "prevTab", "keys": "ctrl+pageup" },
        { "command": "nextTab", "keys": "ctrl+pagedown" }
    ]
```

* Open 3 or more tabs

* Use `ctrl + pageup` or `ctrl + pagedown`, it doesn't jump to  pre or next tab

This settings works well in the version before I upgrade it in Microsoft Store, but I don't know what version was.

# Expected behavior

`ctrl+pageup` to the pre tag

`ctrl+pagedown` to the next tag

# Actual behavior

Random jumps

**Comment by zadjii-msft** [comment]
If you hold <kbd>ctrl</kbd> while you're switching tabs, does a little dialog with a list of your open tabs show up? And does that dialog have them listed in "most recently used" order?

What I'm thinking you're seeing is the new tab switcher which defaults to MRU ordering. You can change this back to in-order traversal with `"tabSwitcherMode": "inOrder"` (or disable it entirely with `"tabSwitcherMode": "disabled"`)

**Comment by WilliamKyle** [comment]
I configured it with `"tabSwitcherMode": "inOrder"` or `"tabSwitcherMode": "disabled"`, both not working in 1.4.3141.0.

And I found `tabSwitcherMode` in https://docs.microsoft.com/en-us/windows/terminal/customize-settings/global-settings, and this is only working in `Windows Terminal Preview`. 

Fortunately `useTabSwitcher` is working in 1.4.3141.0. 
I configured it with `"useTabSwitcher": false` and get what I expected. 

Thanks for your help.

**Comment by davidchisnall** [comment]
For anyone else finding this: It has nothing to do with the key bindings used, the nextTab and prevTab commands are completely broken unless you disable the tab switcher (at least, in v1.5.3142.0).  The tab switcher was annoying when it was first introduced (why do I want a UI to pop up listing tabs right next to another UI element that lists tabs?) and probably should default to off.  It definitely should not default to a behaviour that breaks tab switching.  MRU policies on frequently used UI elements are bad practice in general because they break motor memory.

**Comment by zadjii-msft** [comment]
>  MRU policies on frequently used UI elements are bad practice in general because they break motor memory.

<kbd>alt+tab</kbd> would like to have a word with you

**Comment by davidchisnall** [comment]
There's been a lot of research covering alt-tab and related things for task switching over the last 30 years.  This has led to things like Expose on the Mac and the Tasks View on Windows, which give a stable spatial view that generally do better in user studies than the alt-tab LRU policy, which works well if you are switching between two applications and increasingly badly as that number grows above two.

**Comment by zadjii-msft** [comment]
I'm not gonna disagree with you - I'm an `inOrder` guy myself. But my dedication is to customizability. For everyone out there who likes `inOrder` traversal, there's someone else who prefers `mru`, which is why there are settings to enable both modes. It's IMPOSSIBLE to pick a set of defaults that makes everyone happy, but the goal with the Terminal has always been to enable the user to customize it _just the way they like it_.

Since OP figured this out, I'm gonna close this issue.

**Comment by davidchisnall** [comment]
I don't mind the option of MRU existing, I object to the POLA violation: the meaning of nextTab and prevTab changed during an update and users need to explicitly opt in to the prior behaviour.  The new behaviour is significantly worse because the MRU list is update on every switch, so in a four-tab window you will have one tab that is unreachable using the keyboard shortcuts (this was the symptom that I saw that led me to this bug).

**Comment by zachjweiner** [comment]
FWIW, I was about to post an issue that `"tabSwitcherMode": "inOrder"` does not override the change in default introduced in 1.4.3141.0. (It seems this setting isn't available in 1.4, though?)

So the new tab switcher suddenly only works in MRU mode, and the previous default cannot be restored without disabling the interface entirely. I would think this is a regression, but please let me know if I'm missing something.
